src/main/java/culturemedia/controllers/CultureMediaController.java

```diff
@@ -0,0 +1,26 @@
+package culturemedia.controllers;
+
+import java.util.*;
+
+import culturemedia.exception.VideoNotFoundException;
+import culturemedia.model.Video;
+import culturemedia.service.impl.CultureMediaServiceImpl;
+
```

Comment:
Use **interface** instead of **implementation**. Relying on the interface is preferable, it makes the code more flexible and scalable.